Api/csharp/ArmoniK.Api.Worker/ArmoniK.Api.Worker.csproj

```diff
@@ -46,9 +46,6 @@
     <Protobuf Include="..\..\..\Protos\V1\agent_service.proto" GrpcServices="Both">
       <Link>gRPC\Protos\agent_service.proto</Link>
     </Protobuf>
-    <Protobuf Include="..\..\..\Protos\V1\submitter_service.proto" GrpcServices="Both">
```

Comment:
We need it. It is the packge used to implement this service in core. Unfortunately, it needs to have almost everything to avoid duplicate classes and just conflicts everywhere.